Assets/MixedRealityToolkit/Interfaces/InputSystem/IMixedRealityGazeProvider.cs

```diff
@@ -15,11 +15,6 @@ public interface IMixedRealityGazeProvider
         /// </summary>
         bool Enabled { get; set; }
 
-        /// <summary>
-        /// The <see cref="IMixedRealityInputSystem"/> instance for which this object is providing gaze data.
-        /// </summary>
-        IMixedRealityInputSystem InputSystem { set; }
-
```

Comment:
This is a breaking change as well - thoughts on leaving this obsolete, and then just not using it? Or you guys think we should just axe it and it's okay? 

It feels like there is no customer value here to removing this (it will break them - unless the value is the explicit notification that they need to access the input system separately?)

Reply:
This was a set-only property to get the input system into the gaze provider. Client code relying on this would, in general, cause unexpected effects in the system.

We >could< mark it obsolete, but as it was for internal bookkepping, it felt prudent to remove.

Reply:
We did a cost/benefit analysis and feel this change is the correct thing to do.